apps/modernization-ui/src/apps/page-builder/components/Question/Question.tsx

```diff
@@ -3,23 +3,354 @@ import './Question.scss';
 import { Input } from 'components/FormInputs/Input';
 import { SelectInput } from 'components/FormInputs/SelectInput';
 import { PagesQuestion } from 'apps/page-builder/generated';
+import { Button, ButtonGroup, Grid, Icon, ModalRef, ModalToggleButton, Radio } from '@trussworks/react-uswds';
+import React, { useRef, useState } from 'react';
+import { ModalComponent } from '../../../../components/ModalComponent/ModalComponent';
+import { CreateQuestion } from '../CreateQuestion/CreateQuestion';
+import ReactSelect, { components } from 'react-select';
+import { fieldType, logicList } from '../../constant/constant';
+import { SelectControl } from '../../../../components/FormInputs/SelectControl';
+import { useForm } from 'react-hook-form';
+import { MultiSelectInput } from '../../../../components/selection/multi';
+// import DeleteValueset from '../DeleteValueset/DeleteValueset';
+import { ValuesetLibrary } from '../../pages/ValuesetLibrary/ValuesetLibrary';
 
 export const Question = ({ question }: { question: PagesQuestion }) => {
+    const [qtnId, setQtnId] = useState(0);
+    const editQtn = (id: any) => {
+        setQtnId(id);
+    };
```

Comment:
Issue: This is the `old` question component and should not be modified. The new question component was just created by Alaap in #855